src/exceptions/response-error.ts

```diff
@@ -0,0 +1,15 @@
+import { type ErrorResponse } from '~/types'
+
+class ResponseError extends Error {
+  code?: string
+  status?: number
```

Comment:
why these fields are optional?

Reply:
If we will have error not from our server, we won't know status and code (it will be internal error of Axios or network)